pkg/services/dashboards/database/database.go

```diff
@@ -990,3 +990,14 @@ func (d *DashboardStore) GetDashboardTags(ctx context.Context, query *models.Get
 		return err
 	})
 }
+
+func (d *DashboardStore) HasActivePublicDashboard(ctx context.Context, query *models.HasActivePublicDashboardQuery) error {
+	return d.sqlStore.WithDbSession(ctx, func(dbSession *sqlstore.DBSession) error {
+		sql := "SELECT COUNT(*) FROM dashboard_public WHERE dashboard_uid=? AND is_enabled=1"
+
+		result, err := dbSession.SQL(sql, query.DashboardUid).Count()
+		query.Result = result > 0
```

Comment:
I'm not sure how this exactly works but it seems the `.Count()` counts how many records are returned and in a `select count(*)` there always be one record (the number could be 0 or greater, but one record always).
Do we really need the count in both places? (query and total record results)

Reply:
Eze and I played around with this and apparently you do need the `COUNT` in the raw sql and the `Count()` call on xorm. Makes zero sense 🤷‍♂️

Reply:
We are phasing out of using query models to return values. We should return an error and a boolean instead of returning an object.

Reply:
Cool. I was wondering why all the other ones did it like this. Wasn't able to mock it.